sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -935,6 +935,105 @@ def sdc_pandas_dataframe_drop_impl(df, labels=None, axis=0, index=None, columns=
     return func_def, global_vars
 
 
+def _dataframe_reduce_columns_codegen_isna(func_name, func_params, columns, df):
+    results = []
+    joined = ', '.join(func_params)
+    func_lines = [f'def _df_{func_name}_impl({joined}):']
+    ind = df_index_codegen_all(df)
+    for i, c in enumerate(columns):
+        result_c = f'result_{c}'
+        func_lines += [f'  series_{c} = pandas.Series(get_dataframe_data({func_params[0]}, {i}))',
```

Comment:
```suggestion
        func_lines += [f'  series_{c} = pandas.Series(get_dataframe_data({df}, {i}))',
```
where would be defined above as:
df = func_params[0]